test/test_models.py

```diff
@@ -19,13 +19,19 @@
 
 ACCEPT = os.getenv("EXPECTTEST_ACCEPT", "0") == "1"
 
+# skip big models to reduce memory usage on CI test
+skipped_models = {
+    "vit_h_14",
+    "regnet_y_128gf",
+}
+
 
 def get_models_from_module(module):
     # TODO add a registration mechanism to torchvision.models
     return [
         v
         for k, v in module.__dict__.items()
-        if callable(v) and k[0].lower() == k[0] and k[0] != "_" and k != "get_weight"
+        if callable(v) and k[0].lower() == k[0] and k[0] != "_" and k != "get_weight" and k not in skipped_models
```

Comment:
The specific method is being used in a few places. I'm not sure it's a good idea to remove them from this method. Also note that this one is being used for multiple modules so hardcoding classification models here makes less sense.

Instead we should add this on this skip on the `test_classification_model` method (and potentially in all other methods so that it can be used across all tasks. We should do a proper pytest skip to indicate that the test was skipped and potentially give the option to the user to run it. One common idiom used in TorchVision for such cases is using an env variable (see [this](https://github.com/pytorch/vision/blob/main/test/test_models.py#L20) for example).